azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -611,12 +620,8 @@ public void shutdown() {
    */
   private void shutdownInternal() {
```

Comment:
I'm wondering if this method could be leveraged/called when some severe problem occurs in the executor.